### PR TITLE
feat: update to use faker 7.x -> 8.x

### PR DESCRIPTION
**Pull request by agustif**
WIP

Hey Luis the other day updated your plugin to make it work with faker 9 calls (api changes a bit)

Opening a PR here in case that's something you where thinking of doing in the future

thanks for the plugin really useful to populate some testing environments with synthetic data

### CODE REVIEW

**Comment by luisrudge** [comment]
Thanks @agustif. Can you update the code to put the faker-js package in the dependencies list again? I'll merge it in after that.

File: package.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
     "release": "release-it"
   },
   "dependencies": {
-    "@faker-js/faker": "^7.6.0",
```
Comment (luisrudge) [inline package.json:21]: leave it as a dependency

**Review by luisrudge — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (agustif) [reply package.json:21]: Sure thing luis, done now

**Comment by luisrudge** [comment]
Sorry, I forgot about this :( Seems like the tests are failing because of the new methods.

**Comment by agustif** [comment]
they are snapshot tests no? probably only need to update the snapshot?

**Comment by luisrudge** [comment]
@agustif thanks for the help and patience. Released in 0.11.0

**Comment by agustif** [comment]
> @agustif thanks for the help and patience. Released in 0.11.0

Thank you likewise for your patience with this, should have updated the snapshots tests myself, Github clunky UI not running the tests until you authorize didn't help there but I didn't think to do so locally myself.

Happy that the upgrade got into the latest release!

I had some ideas around doing some kinda based of similarity detection for prisma's model's field names vs faker's available apis, so maybe with just that we would be able to provide better infrence of what fake data to put in each field.

**Comment by luisrudge** [comment]
@agustif I'd love to see what that looks like!

**Pull request merged**